pkg/apis/targets/v1alpha1/register.go

```diff
@@ -42,6 +42,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&AWSComprehendTargetList{},
 		&AWSDynamoDBTarget{},
 		&AWSDynamoDBTargetList{},
+		&AWSEventBridgeTarget{},
+		&AWSEventBridgeTargetList{},
 		&AWSLambdaTarget{},
 		&AWSLambdaTargetList{},
 		&AWSS3Target{},
```

Comment:
While we're at it, would you mind moving the `AWSKinesisTarget` types right below this one?